mmv1/templates/terraform/examples/compute_region_network_firewall_policy_with_rules_full.tf.erb

```diff
@@ -0,0 +1,82 @@
+data "google_project" "project" {
+  provider = google-beta
+}
+
+resource "google_compute_region_network_firewall_policy_with_rules" "<%= ctx[:primary_resource_id] %>" {
+  name        = "<%= ctx[:vars]['policy_name'] %>"
+  region      = "us-west2"
+  description = "Terraform test"
+  provider    = google-beta 
+
+  rule {
+    description    = "tcp rule"
+    priority       = 1000
+    enable_logging = true
+    action         = "allow"
+    direction      = "EGRESS"
+    match {
+      layer4_config {
+        ip_protocol = "tcp"
+        ports       = [8080, 7070]
+      }
+      dest_ip_ranges = ["11.100.0.1/32"]
+      dest_fqdns = ["www.yyy.com", "www.zzz.com"]
+      dest_region_codes = ["HK", "IN"]
+      dest_threat_intelligences = ["iplist-search-engines-crawlers", "iplist-tor-exit-nodes"]
+      dest_address_groups = [google_network_security_address_group.address_group_1.id]
+    }
+    target_secure_tag {
+      name = "tagValues/${google_tags_tag_value.secure_tag_value_1.name}"
+    }
+  }
+  rule {
+      description    = "udp rule"
+      priority       = 2000
+      enable_logging = false
+      action         = "deny"
+      direction      = "INGRESS"
+      match {
+        layer4_config {
+          ip_protocol = "udp"
+        }
+        src_ip_ranges = ["0.0.0.0/0"]
+        src_fqdns = ["www.abc.com", "www.def.com"]
+        src_region_codes = ["US", "CA"]
+        src_threat_intelligences = ["iplist-known-malicious-ips", "iplist-public-clouds"]
+        src_address_groups = [google_network_security_address_group.address_group_1.id]
+        src_secure_tag {
+          name = "tagValues/${google_tags_tag_value.secure_tag_value_1.name}"
+        }
+      }
+      disabled = true
+    }
+}
+
+resource "google_network_security_address_group" "address_group_1" {
+  provider  = google-beta 
+  name        = "address-group-1"
```

Comment:
You'll want to have a test name for security address_group, this is resulting in the failing test below:

```hcl
    vcr_utils.go:167: Step 1/2 error: Error running apply: exit status 1
        
        Error: Error creating AddressGroup: googleapi: Error 409: Resource 'projects/ci-test-project-188019/locations/us-west2/addressGroups/address-group-1' already exists
        Details:
        [
          {
            "@type": "type.googleapis.com/google.rpc.ResourceInfo",
            "resourceName": "projects/ci-test-project-188019/locations/us-west2/addressGroups/address-group-1"
          }
        ]
        
          with google_network_security_address_group.address_group_1,
          on terraform_plugin_test.tf line 56, in resource "google_network_security_address_group" "address_group_1":
          56: resource "google_network_security_address_group" "address_group_1" {
        
        
        Error: Error waiting to create TagKey: Error waiting for Creating TagKey: Error code 6, message: The TagKey namespace 'ci-test-project-188019/tag-key' already exists.
        
          with google_tags_tag_key.secure_tag_key_1,
          on terraform_plugin_test.tf line 67, in resource "google_tags_tag_key" "secure_tag_key_1":
```

Reply:
Done - I fixed the tests.